templates/flutter/base/macros.twig

```diff
@@ -0,0 +1,5 @@
+{% macro parameter(parameter) %}{% if parameter.required %}required {{ parameter.type | typeName }}{% else %}{{ parameter.type | typeName }}?{% endif %} {{ parameter.name | caseCamel | escapeKeyword }}{% endmacro %}
```

Comment:
Do we really need this as an individual template? Can it be just part of the service template? If not, I'm not sure macros is the most intuitive name.

Reply:
It I keep these in service, I can't call the function in `api.twig` or other included template files, keeping this in separate file allows us to import and use it in any template files. so we can define all macros in a separate file and import and use from any file. What might be a better name? `functions` or `utils` ?